modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -186,6 +187,9 @@ else if (Objects.equals(
 				reverseObjectRelationship.getObjectRelationshipId());
 		}
 
+		_objectLayoutTabPersistence.removeByObjectRelationshipId(
```

Comment:
Could we use the method from localService class instead of the persistence?

Reply:
`ObjectLayoutTab` doesn't have LocalService, only Persistence